src/content/docs/tutorials/javareporter/junit.md

```diff
@@ -0,0 +1,156 @@
+</dependency>
+```
+
+```properties
+    implementation group: 'io.testomat', name: 'java-reporter-junit', version: '0.7.4'
+```
+
+> NOTE: There might be an updated version of the library published. You can check it in
+> the [Maven Central Repository](https://central.sonatype.com/artifact/io.testomat/java-reporter-junit)
+
+---
+
+## Importing JUnit tests to [Testomat.io](https://app.testomat.io)
+
+It is important to import your test codebase to [Testomat.io](https://app.testomat.io) to see the correct project
+structure and test source code in the UI.
+
+You can easily do this with the following commands:
+
+Here is the command for UNIX/MAC users:
+
+```bash
+  export TESTOMATIO_URL=... && \
+  export TESTOMATIO=... && \
+  curl -L -O https://github.com/testomatio/java-check-tests/releases/latest/download/testomatio.jar && \
+  java -jar testomatio.jar sync
+```
+
+And here is the command for Windows users:
+
+```cmd
+    set TESTOMATIO_URL=...&& ^
+    set TESTOMATIO=...&& ^
+    curl -L -O https://github.com/testomatio/java-check-tests/releases/latest/download/testomatio.jar&& ^
+    java -jar testomatio.jar sync
+```
+
+This will download the **testomatio.jar** file – a CLI tool that is used to import tests.
+> NOTE: More information about test importing can be found in the related
+> tutorial [Import tests with Java-Check-Tests CLI](java-check-tests.md)
+> and [Java-Check-Tests repository](https://github.com/testomatio/java-check-tests)
+
+---
+
+## Configure Java-Reporter in your project
+
+1. If you don't have the **junit-platform.properties** file in your `resources` folder, create it and add this line:  
+   `junit.jupiter.extensions.autodetection.enabled=true`
+
+   This will activate the service loader mechanism on your project side to allow running the library, as it's based on the JUnit
+   extension.
+   <br/><br/>
+
+2. You will need to provide some properties for reporting.  
+   The list of required properties is quite short:
+    - `testomatio.listening=true` – enables the reporting as a whole.
+    - `testomatio.api.key` – the particular project API key you can obtain in [Testomat.io](https://app.testomat.io) >
+      Account > Access-Tokens
+
+   There are four ways to provide properties:
+    - System environment
+    - JVM properties
+    - `testomatio.properties` file in the resources folder
+    - `testomatio.url=https://beta.testomat.io` – if you run on beta, add this property as the library reports to
+      https://app.testomat.io
+
+   The most reliable and convenient way is to use the properties file.
+
+   > - If you use the **JVM-properties** approach, use the **dot notation** (`testomatio.api.key`)
+
+   > - When using environment variables, write properties in constant case (`TESTOMATIO_API_KEY`)
+
+   > Names of the properties **remain the same** in both approaches.
+
+3. When you have synchronized your code and provided the required properties, that's basically all: run your tests and see
```

Comment:
this is tutorial, not a reference here! Do not focus on all possible options
Focus on usage and workflow